charts/redis-ha/README.md

```diff
@@ -218,6 +218,7 @@ The following table lists the configurable parameters of the Redis chart and the
 | `extraContainers`         | Extra containers to include in StatefulSet                                                                                                                                                               |`[]`|
 | `extraInitContainers`     | Extra init containers to include in StatefulSet                                                                                                                                                          |`[]`|
 | `extraVolumes`            | Extra volumes to include in StatefulSet                                                                                                                                                                  |`[]`|
+| `additionalLabels`        | Labels that should be applied to all created resources                                                                                                                                                   |`{}`|
```

Comment:
Can we make this `extraLabels`? It matches the rest of the chart.

Reply:
Fair suggestion - changes made